jovian/jovian_nb_ext/main.js

```diff
@@ -344,20 +687,20 @@ define([
     );
 
     //toolbar button to remove the jovian extension
-    const remove_ext_action = {
-      icon: "fa-times",
+    const set_paras_ext_action = {
+      icon: "fa-angle-double-down",
       help: "Disable Jovian Extension",
```

Comment:
`help` key is the tooltip for the toolbar button. You can the add the appropriate tooltip here.

Reply:
Yeah! sure, I will change by tomorrow. Thanks for sharing me the way of testing. I will also do that.